crates/config/src/toml.rs

```diff
@@ -1111,7 +1127,7 @@ mod tests {
         assert!(result
             .unwrap_err()
             .to_string()
-            .starts_with("No such file or directory"));
+            .starts_with("no such file or directory"));
```

Comment:
We probably want to additionally have the test confirm the filename is present now, but for the second assertion, use a `contains` check to avoid having to be concerned with whether the path in the error message is relative or absolute.